### DISCUSSION TITLE
Seus Renewed v1.0.1 Weather Glitch on Water (thunder/rain)

### DISCUSSION THREAD

**Discussion by gamerpotion** [IrisShaders/Iris#950] [Q&A] [answered]
Hi, sorry if this is known already.  Seus Renewed works well now on 1.1.2, however, I have noticed when the weather is set to rain or thunder the water starts to glitch out - [https://youtu.be/YHjmkja--eg](url)
Any reason for what could be causing this?

**Comment by IMS212** [comment, marked as answer]
This is due to custom textures not being supported, which will hopefully be fixed in the next release.

**Reply by gamerpotion** [reply]
Thank you, you were right!  I tested  `#define RAIN_SPLASH_EFFECT` to `//#define RAIN_SPLASH_EFFECT` on water & terrain it's not doing that glitch now [https://youtu.be/Cw2mqbxhDWA](https://youtu.be/Cw2mqbxhDWA)
Can't wait to try out the new Iris release :)
Many Thanks @IMS212
